features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/system-api-resource.xml.j2

```diff
@@ -839,6 +839,34 @@
                     description="Delete OAuth2 scopes in the organization (root)"/>
         </Scopes>
     </APIResource>
+    <APIResource name="Certificate Validation Management API" identifier="/api/server/v1/certificate-validation"
```

Comment:
Let's update xml files also for consistency

Reply:
fixed with https://github.com/wso2/carbon-identity-framework/pull/6448/commits/f5ee93c59a2b06cf45fd9151d5b3deab0bdf59c1